test/functionality/r-bridge/lang/ast/parse-values.ts

```diff
@@ -29,7 +29,7 @@ describe('Constant Parsing',
 					request:                'text',
 					content:                '{',
 					ensurePackageInstalled: true
-				}, shell))
+				}, shell) as Promise<string>)
```

Comment:
without being able to track the context in GitHub (thanks GitHub) how about [AsyncOrSync](https://github.com/ts-essentials/ts-essentials/tree/master/lib/async-or-sync)?